### PR TITLE
Extended the transformations to include .on event observers

### PR DESCRIPTION
**Pull request by j-carvalho**
Ensure .on prototype extensions is converted to Ember.on ...

### CODE REVIEW

**Comment by kamal** [comment]
lgtm

File: tests/fixtures/prototype-extension-files/new.js [hunk 1]
```diff
@@ -27,5 +27,9 @@ export default Ember.Controller.extend({
 
   chainedObserver: Ember.observer('foo', 'bar', function(property) {
     this.set('baz', true);
-  }).on('init')
+  }).on('init'),
```
Comment (stefanpenner) [inline tests/fixtures/prototype-extension-files/new.js]: this is actually wrong, its now `Ember.on(Ember.observer(..`

**Comment by abuiles** [comment]
@j-carvalho thanks!  I'm currently traveling so won't be able to fully look at this until tomorrow/sunday morning, can you please address @stefanpenner comment on the meantime?

File: tests/fixtures/prototype-extension-files/new.js (hunk 1 above)
Comment (j-carvalho) [inline tests/fixtures/prototype-extension-files/new.js]: You mean `Ember.on('init', Ember.observer(...` right?

**Comment by j-carvalho** [comment]
Fixed the issue @abuiles. @stefanpenner can you please confirm if it's now correct?

**Comment by stefanpenner** [comment]
yes.

**Pull request merged**